src/Sulu/Bundle/AdminBundle/DependencyInjection/Compiler/AddAdminPass.php

```diff
@@ -19,7 +19,7 @@
  */
 class AddAdminPass implements CompilerPassInterface
 {
-    const ADMIN_TAG = 'sulu.admin';
```

Comment:
this change is not compatible with php 5.6. not sure if it is possible to configure `php-cs-fixer` to be compatible with older php versions

Reply:
okay looks like `php-cs-fixer` will target the php version that is used for running it. as the new `php-cs-fixer` version is not compatible with php versions smaller than 7.1, we cannot update the `1.6` branch 😕